Source/FakeItEasy.Specs/UnconfiguredFakeSpecs.cs

```diff
@@ -6,23 +6,28 @@
 
     public static class UnconfiguredFakeSpecs
     {
+        public interface IFoo
+        {
+            bool IsADummy { get; }
+        }
+
         [Scenario]
         public static void VirtualMethod(
             MakesVirtualCallInConstructor fake)
         {
-            "when faking a class with a virtual method"
+            "When faking a class with a virtual method"
                 .x(() => fake = A.Fake<MakesVirtualCallInConstructor>());
 
-            "it should return a default value when the method is called during the constructor"
+            "It should return a default value when the method is called during the constructor"

```

Comment:
I don't mind too much either way. Toss a coin :wink: 

Why do you feel the need to include "unconfigured"? Because there is no `FakeConfigurator` hiding somewhere for it?